vta/hardware/chisel/src/main/scala/core/TensorGemm.scala

```diff
@@ -126,8 +145,7 @@ class MatrixVectorMultiplication(implicit p: Parameters) extends Module {
   })
   val dot = Seq.fill(size)(
     Module(new DotProduct(aBits = inpBits, bBits = wgtBits, size)))
-  val acc = Seq.fill(size)(
-    Module(new Pipe(UInt(accBits.W), latency = log2Ceil(size) + 1)))
+  val acc = Seq.fill(size)(Module(new Pipe(UInt(accBits.W), latency = 2)))
```

Comment:
do you mind explaining what this hard coded latency represents?

Reply:
It's one cycle for MAC module (, which is a fused-mulitply-adder (FMA)), the other one cycle for one `PipeAdder` in the first layer of the accumulator. Therefore, this 1+1=2 should be smaller than 4 (the states iterate over `sReadUop :: sComputeIdx :: sReadTensor :: sExe` in TensorGemm module), in order to ensure accumulated `acc` should be available for sReadTensor stage. Does this make sense?

Reply:
Thanks for the explanation; totally makes sense. Can you add what you explained as a code comment?

Reply:
comments added. (see L155 and L218 in the file.)